### PR TITLE
Updating intercom integration

### PR DESCRIPTION
**Pull request by Tolsa**
- add Intercom integration to Rails/Lumber
- update intercom integration in Sequelize

### CODE REVIEW

File: _includes/express-sequelize/integrations.md [hunk 1]
```diff
@@ -25,17 +25,15 @@ app.use(Liana.init({
...
+      mapping: ['user'],
     }
   }
 }));
 ```
 
-<div class="c-notice warning l-mt">
-  ⚠️ The integration needs an "extended" Intercom Access token to let the Forest liana read users and conversations.
-</div>
+The "mapping" points the Forest collection you want to map the data coming from Intercom with.
```
Comment (louisteenbrink) [inline _includes/express-sequelize/integrations.md]: The "mapping" option is the collection on which your integration is pointing.

File: _includes/lumber/integrations.md [hunk 2]
```diff
@@ -46,3 +46,19 @@ If you want to add other fields from Mixpanel, you have to add them in `customPr
 You will then be able to see the Mixpanel events on a record, a `Customer` in our example.
 
 ![Mixpanel integration 1](/public/img/mixpanel1.png "mixpanel-integration")
+
+## Intercom
+
+<span class="l-category l-category__pro">Premium feature</span>
+
+Configuring the Intercom integration for Forest allows you to have your user's
```
Comment (louisteenbrink) [inline _includes/lumber/integrations.md]: Configuring the Intercom integration for Forest allows you to display your user's session data (location, browser type, ...) and conversations.

You have to add the Intercom integration in your `app.js` file, as described below. The "mapping" option is the collection on which your integration is pointing.

File: _includes/rails/integrations.md [hunk 3]
```diff
@@ -46,4 +46,20 @@ You will then be able to see the Mixpanel events on a record, a `Customer` in ou
 
 ![Mixpanel integration 1](/public/img/mixpanel1.png "mixpanel-integration")
 
+## Intercom
+
+<span class="l-category l-category__pro">Premium feature</span>
+
+Configuring the Intercom integration for Forest allows you to have your user's
```
Comment (louisteenbrink) [inline _includes/rails/integrations.md]: Configuring the Intercom integration for Forest allows you to display your user's session data (location, browser type, ...) and conversations.

You have to add the Intercom integration in your `app.js` file, as described below. The "mapping" option is the collection on which your integration is pointing.

Comment (Tolsa) [reply _includes/rails/integrations.md]: ok ;) t'as copy paste la 2eme partie mais j'adapte évidemment (la partie avec app.js ne s'applique pas à rails) @louisteenbrink

**Pull request merged**